pages/common/gixy.md

```diff
@@ -0,0 +1,20 @@
+# gixy
+
+> Analyze Nginx configuration.
+> More information: <https://github.com/yandex/gixy>.
+
+- Analyze nginx configuration (default path: /etc/nginx/nginx.conf):
+
+`gixy`
+
+- Analyze nginx configuration but skip specific tests:
+
+`gixy --skips {{http_splitting}}`
+
+- Analyze nginx configuration with the specific level:
+
+`gixy {{-l|-ll|-lll}}`
+
+- Analyze nginx configuration on the specific path:
+
+`gixy {{config_path}}`
```

Comment:
```suggestion
`gixy {{path/to/configuration_file}}`
```
Unless it's a config directory, in which case `configuration_directory` :slightly_smiling_face: